common/src/main/kotlin/pl/edu/prz/kod/common/application/EnvironmentVariable.kt

```diff
@@ -0,0 +1,25 @@
+package pl.edu.prz.kod.common.application
```

Comment:
I'm not convinced that this class should be moved to commons - these variable are relevant only for mediator service.

Reply:
I'd propose moving to https://github.com/config4k/config4k soon so the whole configuration should be simplified.

Reply:
I must have moved it accidentally, of course it was supposed to be in mediator. Fixed now.